doc/2/framework/classes/backend-config/properties/index.md

```diff
@@ -9,14 +9,26 @@ description: BackendConfig class properties
 
 <SinceBadge version="2.8.0" />
```

Comment:
Can you add `auto-version` ?

Reply:
The modification was available since the beginning, the only new feature here is that it is typed now